appstudio-utils/Dockerfile

```diff
@@ -1,8 +1,10 @@
 FROM quay.io/enterprise-contract/cli:latest AS ec-image
+FROM quay.io/konflux-ci/buildah-task:latest AS buildah-task-image
 FROM registry.access.redhat.com/ubi9/ubi
 
 COPY --from=ec-image /usr/local/bin/ec_linux_amd64.gz /usr/bin/ec.gz
 RUN gunzip /usr/bin/ec.gz && chmod +x /usr/bin/ec
+COPY --from=buildah-task-image /usr/bin/retry-func.sh /usr/bin/
```

Comment:
`quay.io/konflux-ci/buildah-task:latest@sha256:4d8273444b0f2781264c232e12e88449bbf078c99e3da2a7f6dcaaf27bc53712` does not have `retry-func.sh`.

Other options:

* quay.io/konflux-ci/oras
* quay.io/konflux-ci/konflux-test
* `retry` function in `hack/build-and-push.sh`

Reply:
requires https://github.com/konflux-ci/buildah-container/pull/139 to be merged and image built